webdriver/result.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+class Result():
+    def __init__(self, response):
+        self.status = response.status
+        self.body = response.read()
+
+        if self.body:
+            self.body = json.loads(self.body)
+
+        self._validate(response)
+
+    def __repr__(self):
+        return 'wdclient.Result(status=%d, body=%s)' % (self.status, self.body)
+
+    # SpecID: dfn-send-a-response
+    def _validate(self, response):
+        # > 3. Set the response's header with name and value with the following
+        # >    values:
+        # >
+        # >    "Content-Type"
+        # >       "application/json; charset=utf-8"
+        # >    "cache-control"
+        # >       "no-cache"
+        assert response.getheader("Content-Type") == "application/json; charset=utf-8"
+        assert response.getheader("cache-control") == "no-cache"
```

Comment:
Character case doesn't actually matter for HTTP headers, and [the standard library accounts for
this](https://hg.python.org/cpython/file/2.7/Lib/rfc822.py#l297). I was on the fence about using the idiomatic spelling versus mirroring the spec, so I'm glad you made them one and the same!